styleguide/source/assets/scss/01-atoms/_links.scss

```diff
@@ -17,6 +17,15 @@ a {
   }
 }
 
+aside {
```

Comment:
This limits the scope of links not having a hover background to only the asides. What if we have links in the masthead or below the content?

I would recommend doing something a bit more generic like adding the ama__link--black class to the links to the aside links because they don't have background colors on hover.

Reply:
@designerneil good question -- But it would seem to me that these dedicated `sections` would have **dedicated** styles. e.g. `ama__masthead__content__link` and maybe a future -- `ama__footer__content__link, right?

Reply:
Perhaps we can just put the hover on __page-content?

Reply:
The rule is only have links with purple hover states on page content. Perhaps we should limit the hover states to the class page-content?

Reply:
@designerneil sounds good.